### ISSUE TITLE
UI for fanvid viewing

### ISSUE THREAD

**Issue by sandalwoodbox** [FanvidDB/fanviddb-server#20]
It would be nice to have a way to "share" fanvids with unauthenticated users via a public viewing UI. Blocked by https://github.com/FanvidDB/fanviddb-server/issues/19

**Comment by sandalwoodbox** [comment]
There is already a public viewing UI but we could add a "share" link.
